### ISSUE TITLE
Add mention of retain support

### ISSUE THREAD

**Issue by tsturzl** [streamnative/mop#601]
**Is your enhancement request related to a problem? Please describe.**
Retain semantics were added, however I think it would be useful to mention the support for this and any of the caveats related to this feature. One of the listed caveats is that standalone clusters hold retained messages in memory and they are lost on restart. This might be worth putting into the README.md. This is the [PR for the features](https://github.com/streamnative/mop/pull/581).

**Describe the solution you'd like**
Add mention of the retain support in MOP, and also mention any known caveats and maybe brief notes on behavior in the README

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

**Comment by Technoboy-** [comment]
Yes, right. Thanks @tsturzl . Could you help push a patch for this ?
And how about putting it here ?
![image](https://user-images.githubusercontent.com/6297296/165213933-f33472f6-b715-4530-bf1a-2084abc1b152.png)

**Comment by tsturzl** [comment]
I'd like to add this, but frankly I'm not fully informed on the implementation details of this feature because I did not contribute to it. I can add the aforementioned caveat that was mentioned in my original post, but that's about all I know in regards to that.

Some things I've wondered about this feature is if a MQTT subscriber connects, is there a way to publish to the topic from a pulsar client in a way that retains the message in MoP? MQTT retain is only set on publish, so I wonder also what the behavior would be in the case that you publish a retained message on MQTT and then subsequently publish a message through pulsar on that same topic, does it continue to retain the message that is technically older? I think it would be worth mentioning whether a Pulsar message has any effects on the retain mechanism, and currently I assume it doesn't.
